juniper_axum/examples/simple.rs

```diff
@@ -29,34 +31,34 @@ impl Query {
     }
 }
 
+#[derive(Clone, Copy, Debug)]
 pub struct Subscription;
+
 type NumberStream = Pin<Box<dyn Stream<Item = Result<i32, FieldError>> + Send>>;
-type AppSchema = RootNode<'static, Query, EmptyMutation<Context>, Subscription>;
 
 #[graphql_subscription(context = Context)]
 impl Subscription {
     /// Count seconds
     async fn count() -> NumberStream {
         let mut value = 0;
-        let stream = tokio_stream::wrappers::IntervalStream::new(tokio::time::interval(
-            Duration::from_secs(1),
-        ))
-        .map(move |_| {
+        let stream = IntervalStream::new(interval(Duration::from_secs(1))).map(move |_| {
             value += 1;
             Ok(value)
         });
         Box::pin(stream)
     }
 }
 
+type AppSchema = RootNode<'static, Query, EmptyMutation<Context>, Subscription>;
+
 #[tokio::main]
 async fn main() {
-    let schema = Arc::new(AppSchema::new(Query, EmptyMutation::new(), Subscription));
+    let schema = AppSchema::new(Query, EmptyMutation::new(), Subscription);
 
     let context = Context;
 
     let app = Router::new()
-        .route("/", get(|| playground("/graphql", Some("/subscriptions"))))
```

Comment:
@btielen Great job, thanks! I've made some insignificant changes though. Most notably simplifying the usage of the `playground`, now it feels to me a lot more `axum`y.